include/awkward/array/RawArray.h

```diff
@@ -25,7 +25,8 @@
 #include "awkward/array/IndexedArray.h"
 
 namespace awkward {
-  void tojson_boolean(ToJson& builder, bool* array, int64_t length) {
+  // function included in multiple source files must be inline
+  inline void tojson_boolean(ToJson& builder, bool* array, int64_t length) {
```

Comment:
`inline` here is a linkage directive. The function's definition showed up in multiple translation units when I tested usage of a `RawArray` in other arrays. Eventually, I used other implementations that did not require a `RawArray`, but was not sure if I should roll back the `inline`.

Here is the compiler I'm using:
```
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin19.2.0
Thread model: posix
```

Reply:
Okay, if it avoids unnecessary linker duplication, then sure. I was thinking that it was a compiler directive, intended to avoid creating a separate function. I didn't know about the other definition.